app/helpers/shop_helper.rb

```diff
@@ -52,4 +52,10 @@ def show_shopping_cta?
 
     true
   end
+
+  private
+
+  def show_groups_tabs
```

Comment:
Ruby methods usually end with `?` if the return value is boolean.
```suggestion
  def show_groups_tabs?
```